spec/registry.md

```diff
@@ -255,3 +255,548 @@ which only expects the `accord` option:
+- `useGrouping`
+  - `auto` (default)
+  - `always`
+  - `min2`
+- `minimumIntegerDigits`
+  - (non-negative integer, default: `1`)
+- `maximumSignificantDigits`
+  - (non-negative integer)
+
+> [!NOTE]
+> The following options and option values are being developed during the Technical Preview
+> period.
+
+The following values for the option `style` are _not_ part of the default registry.
+Implementations SHOULD avoid creating options that conflict with these, but
+are encouraged to track development of these options during Tech Preview:
+- `currency`
+- `unit`
+
+The following options are _not_ part of the default registry.
+Implementations SHOULD avoid creating options that conflict with these, but
+are encouraged to track development of these options during Tech Preview:
+- `currency`
+   - valid [Unicode Currency Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeCurrencyIdentifier)
+     (no default)
+- `currencyDisplay`
+   - `symbol` (default)
+   - `narrowSymbol`
+   - `code`
+   - `name`
+- `currencySign`
+  - `accounting`
+  - `standard` (default)
+- `unit`
+   - (anything not empty)
+- `unitDisplay`
+   - `long`
+   - `short` (default)
+   - `narrow`
+
+#### Default Value of `select` Option
+
+The value `plural` is default for the option `select` 
+because it is the most common use case for numeric selection.
+It can be used for exact value matches but also allows for the grammatical needs of 
+languages using CLDR's plural rules.
+This might not be noticeable in the source language (particularly English), 
+but can cause problems in target locales that the original developer is not considering.
+
+For example, a naive developer might use a special message for the value `1` without
+considering a locale's need for a `one` plural:
+```
+.match {$var}
+1   {{You have one last chance}}
+one {{You have {$var} chance remaining}} // needed by languages such as Polish or Russian
+                                         // such locales typically require other keywords
+                                         // such as two, few, many, and so forth
+*   {{You have {$var} chances remaining}}
+```
+
+#### Percent Style
+When implementing `style=percent`, the numeric value of the _operand_ 
+MUST be multiplied by 100 for the purposes of formatting.
+
+> For example,
+> ```
+> The total was {0.5 :number style=percent}.
+> ```
+> should format in a manner similar to:
+> > The total was 50%.
+
+### Selection
+
+Number selection has three modes:
+- `exact` selection matches the operand to explicit numeric keys exactly
+- `plural` selection matches the operand to explicit numeric keys exactly
+  or to plural rule categories if there is no explicit match
+- `ordinal` selection matches the operand to explicit numeric keys exactly
+  or to ordinal rule categories if there is no explicit match
+
+When implementing [`MatchSelectorKeys`](formatting.md#resolve-preferences), 
```

Comment:
Same general suggestions as the ones I made for `:string` (name the arguments here, then refer to them below)